subprojects/groovy-json/build.gradle

```diff
@@ -21,13 +21,19 @@ plugins {
 }
 
 dependencies {
-    api rootProject  // JsonBuilder extends GroovyObjectSupport...
```

Comment:
I think it is useful to keep the comment, otherwise it is hard to track down why we use `api` here and not `implementation`. We should have a similar comment for each api dependency.

Reply:
comment restored